core/admin-login-verify.php

```diff
@@ -35,9 +35,10 @@
 
 
     		else{		    
-              
+				$name = $user['Name'];
```

Comment:
@vishnuk7 why this?

Reply:
@prateek76 I used name variable here
![image](https://user-images.githubusercontent.com/31013847/52525498-ab48b780-2cd0-11e9-8b2c-022ee864b90a.png)

I know there is not need to write this, instead I can write this 	$_SESSION['Name']=$user['Name'];